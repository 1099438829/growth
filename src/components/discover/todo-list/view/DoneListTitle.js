import React, { Component } from 'react';
import { View, Text } from 'react-native';
import AppStyle from '../../../../theme/styles';

class DoneListTitle extends Component {
  static componentName = 'DoneListTitle';
  render() {
    return (
      <View
        style={AppStyle.titleVerticalCenterColorGrayStyle}
      >
        <Text style={{ color: '#999' }}>已完成</Text>
      </View>);
  }
}
export default DoneListTitle;
