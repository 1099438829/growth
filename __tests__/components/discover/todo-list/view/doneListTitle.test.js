import 'react-native';
import React, {} from 'react';
import renderer from 'react-test-renderer';
import DoneListTitle from '../../../../../src/components/discover/todo-list/view/DoneListTitle';

it('renders correctly', () => {
  const doneListTitle = renderer.create(
    <DoneListTitle />,
  );
  const doneListTitleJson = doneListTitle.toJSON();
  expect(doneListTitleJson).toMatchSnapshot();
});
