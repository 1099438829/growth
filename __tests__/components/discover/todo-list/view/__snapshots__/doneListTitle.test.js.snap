// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(245,245,245)",
      "height": 40,
      "justifyContent": "center",
      "paddingLeft": 15,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#999",
      }
    }
  >
    已完成
  </Text>
</View>
`;
